controllers/Course.py

```diff
@@ -33,4 +34,38 @@ def handleAddCollaborator(self, collaborator):
 
     def handleRemoveCollaborator(self, removeCollaborator):
         self.service.removeCollaborator(removeCollaborator)
-        return {"message": "Collaborator correctly removed", "status": status.HTTP_200_OK}
+        return {
+            "message": "Collaborator correctly removed",
+            "status": status.HTTP_200_OK,
+        }
+
+    def handleAddSubscriber(self, courseId, subscriberId):
+        self.service.addSubscriber(courseId, subscriberId)
+        return {"message": "Successful subscription", "status": status.HTTP_200_OK}
+
+    def handleRemoveSubscriber(self, courseId, subscriberId):
+        self.service.removeSubscriber(courseId, subscriberId)
+        return {"message": "Successful unsubscription", "status": status.HTTP_200_OK}
+
+    def handleGetMyCourses(self, userId):
+        # filter = {
```

Comment:
No comenten lineas, mejor borrarlas

Reply:
Era para no olvidarme mi idea, just that